feg/radius/src/modules/module_test_utils.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modules
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"layeh.com/radius"
+)
+
+func WaitForRadiusServerToBeReady(secret []byte, port string) (err error) {
+	MaxRetries := 10
+	for r := 0; r < MaxRetries; r++ {
+		_, err = radius.Exchange(context.Background(), radius.New(radius.CodeStatusServer, secret), port)
+		if err == nil {
+			return nil
+		}
+	}
+	return errors.New(fmt.Sprintf("radius server failed to be ready after %d retries: %v", MaxRetries, err))
```

Comment:
Some tests fail with this error message for me. Do we need to add some kind of sleep statement to the retries?

Reply:
In my testing replacing this function with a 10ms sleep worked as well, but static sleeps are not really good practice. Maybe I can change the timeout to a time limit of 50ms or so instead of using a limited number of retries.

Reply:
For me it always worked before, but I get the same error now. The problem was that for the server address, the port number must come after the last colon, i.e. if it's just a port it must be prepended with a colon. I fixed it and the unit tests should now work again.